wynnset/tapestry-wp
# 418 Display GF Images In Previous Activity and Summary - TYDE

TYDE PR for #425 
Closes Issue #418 

Adds:
- Displaying of new checklist entry object to TydeActivitySummary component

Steps to Test:
- Answer an GF Image Choice checklist Activity and submit the answer
- Add the Activity to profile and check that the selected image displays correctly
- Find the associated Module in the spaceship cockpit and view the activity summary 
- Verify image renders correctly there as well (It should because it is the same component in both locations)
